src/events/callback/handler.cpp

```diff
@@ -11,6 +11,6 @@ namespace Bald {
     Handler::~Handler() {
         auto iter = std::find(m_TakenID.begin(), m_TakenID.end(), m_ID);
         m_TakenID.erase(iter);
+        CORE_LOG_INFO("[Handler] Successfully removed current ID");
```

Comment:
This is a destructor, I think something like "Successfully destroyed handler object" would make more sense.